repos/fdbt-site/tests/pages/selectPassengerType.test.tsx

```diff
@@ -106,14 +108,21 @@ describe('pages', () => {
                     csrfToken="csrf"
                     savedGroups={[]}
                     savedPassengerTypes={savedPassengerTypes}
+                    selectedId={null}
```

Comment:
Do we have a case where this is set?

Reply:
https://github.com/fares-data-build-tool/create-data/pull/177/commits/916e1bbdd8a0ccdc80455d8c6757bcca9a5922d6

Reply:
Done